client/service/test/src/client/light.rs

```diff
@@ -686,7 +686,7 @@ fn changes_proof_is_generated_and_checked_when_headers_are_not_pruned() {
 		match local_result == expected_result {
 			true => (),
 			false => panic!(format!("Failed test {}: local = {:?}, expected = {:?}",
-			                        index, local_result, expected_result)),
+									index, local_result, expected_result)),
```

Comment:
I don't think these adhere to the style guide, but anyhow that's a different story 😄